src/main/java/seedu/address/logic/commands/route/CreateRouteCommand.java

```diff
@@ -16,28 +23,39 @@
 
     public static final String COMMAND_WORD = "create";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Creates a route with a destination. "
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Creates a route with a set of orders. "
             + "Parameters: "
-            + PREFIX_ADDRESS + "ADDRESS\n"
+            + PREFIX_ORDER + "ORDER_ID\n"
             + "Example: " + RouteCommand.COMMAND_WORD + " " + COMMAND_WORD + " "
-            + PREFIX_ADDRESS + "311, Clementi Ave 2, #02-25 ";
+            + PREFIX_ORDER + "1 " + PREFIX_ORDER + "3";
 
     public static final String MESSAGE_SUCCESS = "Route created.";
     public static final String MESSAGE_DUPLICATE_ROUTE = "This route already exists in the address book";
 
-    private final Route toAdd;
+    private final Set<Index> orderIds;
 
     /**
      * Creates an AddCommand to add the specified {@code Route}
      */
-    public CreateRouteCommand(Route route) {
-        requireNonNull(route);
-        toAdd = route;
+    public CreateRouteCommand(Set<Index> orderIds) {
+        requireNonNull(orderIds);
+        this.orderIds = orderIds;
     }
 
     @Override
     public CommandResult execute(Model model, CommandHistory history) throws CommandException {
         requireNonNull(model);
+        List<Order> lastShownList = model.getFilteredOrderList();
+        Set<Order> ordersToAdd = new HashSet<>();
+
+        for (Index id : orderIds) {
+            if (id.getZeroBased() >= lastShownList.size()) {
+                throw new CommandException(Messages.MESSAGE_INVALID_ORDER_DISPLAYED_INDEX);
```

Comment:
Rather than one wrong order causing the whole command to cause an `Exception`, why not maintain a list of successful order additions and unsuccessful ones, and show an "outcome" of the execution -- successful & unsuccessful orders

Reply:
Great idea, changing it now 👍